euclidlib/photo/_phz.py

```diff
@@ -86,7 +86,7 @@ def redshift_distributions(
 def _(
     path: str | PathLike[str],
     z: ArrayLike,
-    nz: ArrayLike,
+    nz: dict[int, NDArray[Any]],
```

Comment:
Wouldn't it make sense for the format of the photo-z to follow that of the maps? i.e. ('POS', 1), ('SHE', 2), etc